modules/widgets/contact-info.php

```diff
@@ -293,7 +293,7 @@ public function form( $instance ) {
 					echo $this->build_map( $instance['address'], $apikey ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 				} elseif ( true !== $instance['goodmap'] && ! empty( $instance['goodmap'] ) ) {
 					printf(
-						'<span class="button-link-delete">%s</span>',
+						'<span class="notice notice-warning" style="display: block;">%s</span>',
```

Comment:
Is this change intended? It seems to be unrelated to this block

Reply:
Looks like it happened due to rebase with an outdated master. I've restore the file to it's current master state via

```
git checkout origin/master -- modules/widgets/contact-info.php
```

I also compared against the file on Github

https://github.com/Automattic/jetpack/blob/master/modules/widgets/contact-info.php